libs/domain/search/facet-rating/facet-rating.component.ts

```diff
@@ -14,14 +14,14 @@ export class SearchRatingFacetComponent extends SearchFacetComponent {
     facetValue: FacetValue
   ): TemplateResult {
     return html`
-      <div class="rating-facet-label">
```

Comment:
We can't change html of the existing component so easily as long as it can lead to a breaking change.

Reply:
But it is a new component. Maybe you mean the same change in `color facet` component?

Reply:
Yep, exactly.